Tests failing on Windows because of dtype differences.

Test summary:

```
========================================================================================================= short test summary info ==========================================================================================================
FAILED tests/summarizing/test_compute_count_summary.py::test_groupby_deployment - AssertionError: Attributes of DataFrame.iloc[:, 1] (column name="total_images") are different
FAILED tests/summarizing/test_compute_count_summary.py::test_groupby_location - AssertionError: Attributes of DataFrame.iloc[:, 1] (column name="total_images") are different
FAILED tests/summarizing/test_compute_count_summary.py::test_add_records_by_class_deployment - AssertionError: Attributes of DataFrame.iloc[:, 1] (column name="total_images") are different
FAILED tests/summarizing/test_compute_count_summary.py::test_add_records_by_class_location - AssertionError: Attributes of DataFrame.iloc[:, 1] (column name="total_images") are different
FAILED tests/summarizing/test_compute_count_summary.py::test_add_species_by_class_deployment - AssertionError: Attributes of DataFrame.iloc[:, 1] (column name="total_images") are different
FAILED tests/summarizing/test_compute_count_summary.py::test_add_species_by_class_location - AssertionError: Attributes of DataFrame.iloc[:, 1] (column name="total_images") are different
FAILED tests/summarizing/test_compute_count_summary.py::test_remove_unidentified_kws - AssertionError: Attributes of DataFrame.iloc[:, 1] (column name="total_images") are different
FAILED tests/summarizing/test_compute_count_summary.py::test_remove_duplicates_kws - AssertionError: Attributes of DataFrame.iloc[:, 1] (column name="total_images") are different
====================================================================================================== 8 failed, 132 passed in 9.68s =======================================================================================================
```

Error raised:
```
E       AssertionError: Attributes of DataFrame.iloc[:, 1] (column name="total_images") are different
E
E       Attribute "dtype" are different
E       [left]:  int32
E       [right]: int64
```

--- a/tests/summarizing/test_compute_count_summary.py
+++ b/tests/summarizing/test_compute_count_summary.py
@@ -110,7 +110,7 @@ def test_groupby_deployment(images):
             "taxa": [2, 5],
         }
     )
-    pd.testing.assert_frame_equal(result, expected)
+    pd.testing.assert_frame_equal(result, expected, check_dtype=False)
 
 
 def test_groupby_location(images, deployments):
@@ -124,7 +124,7 @@ def test_groupby_location(images, deployments):
             "taxa": [6],
         }
     )
-    pd.testing.assert_frame_equal(result, expected)
+    pd.testing.assert_frame_equal(result, expected, check_dtype=False)
 
 
 def test_add_records_by_class_deployment(images):
@@ -142,7 +142,7 @@ def test_add_records_by_class_deployment(images):
             "taxa": [2, 5],
         }
     )
-    pd.testing.assert_frame_equal(result, expected)
+    pd.testing.assert_frame_equal(result, expected, check_dtype=False)
 
 
 def test_add_records_by_class_location(images, deployments):
@@ -160,7 +160,7 @@ def test_add_records_by_class_location(images, deployments):
             "taxa": [6],
         }
     )
-    pd.testing.assert_frame_equal(result, expected)
+    pd.testing.assert_frame_equal(result, expected, check_dtype=False)
 
 
 def test_add_species_by_class_deployment(images):
@@ -178,7 +178,7 @@ def test_add_species_by_class_deployment(images):
             "taxa_aves": [0, 3],
         }
     )
-    pd.testing.assert_frame_equal(result, expected)
+    pd.testing.assert_frame_equal(result, expected, check_dtype=False)
 
 
 def test_add_species_by_class_location(images, deployments):
@@ -196,7 +196,7 @@ def test_add_species_by_class_location(images, deployments):
             "taxa_aves": [3],
         }
     )
-    pd.testing.assert_frame_equal(result, expected)
+    pd.testing.assert_frame_equal(result, expected, check_dtype=False)
 
 
 def test_remove_unidentified_kws(images):
@@ -210,7 +210,7 @@ def test_remove_unidentified_kws(images):
             "taxa": [1, 4],
         }
     )
-    pd.testing.assert_frame_equal(result, expected)
+    pd.testing.assert_frame_equal(result, expected, check_dtype=False)
 
 
 def test_remove_duplicates_kws(images):
@@ -226,7 +226,7 @@ def test_remove_duplicates_kws(images):
             "taxa": [2, 5],
         }
     )
-    pd.testing.assert_frame_equal(result, expected)
+    pd.testing.assert_frame_equal(result, expected, check_dtype=False)
 
 
 def test_invalid_groupby(images, deployments):

